src/queue.hpp

```diff
@@ -727,11 +800,44 @@ struct cvk_command_batch : public cvk_command {
 
     cl_uint batch_size() { return m_commands.size(); }
 
-    bool is_profiled_by_executor() const override final { return false; }
+    CHECK_RETURN cl_int
+    set_profiling_info(cl_profiling_info pinfo) override final {
+        cl_int status = cvk_command::set_profiling_info(pinfo);
```

Comment:
This means batch commands never use the profiling info captured on the device. I think that's fine.